gemrb/core/IniSpawn.cpp

```diff
@@ -729,7 +729,9 @@ void IniSpawn::SpawnCreature(const CritterEntry& critter) const
 	cre->SetPosition(critter.SpawnPoint, true);
 	cre->SetOrientation(ClampToOrientation(critter.Orientation), false);
 
-	cre->SetScriptName(critter.ScriptName);
+	if (!critter.ScriptName.empty()) {
+		cre->SetScriptName(critter.ScriptName);
+	}
```

Comment:
Please add a comment explaining why this is needed and the test case, since it's not really intuitive (and as such at risk of being removed later down the line).